### PR TITLE
eslint

### PR DESCRIPTION
**Pull request by Earlo**

### CODE REVIEW

File: config/middlewares.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  'strapi::security',
+  'strapi::cors',
+  'strapi::poweredBy',
+  'strapi::logger',
+  'strapi::query',
+  'strapi::body',
+  'strapi::session',
+  'strapi::favicon',
+  'strapi::public',
+]
```
Comment (PurkkaKoodari) [inline config/middlewares.js:12]: Is this Strapi v4 related or should it be in this PR?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "parser": "babel-eslint",
+  "extends": "eslint:recommended",
```
Comment (PurkkaKoodari) [inline .eslintrc:3]: We're using `eslint-config-airbnb` in some other repos. Is that implementable with reasonable effort or does Strapi mess it up?

File: config/middlewares.js (hunk 1 above)
Comment (Earlo) [reply config/middlewares.js:12]: Yes

**Review by lakaniemi — APPROVED** [review]
